tests/phpunit/integration/Modules/Analytics_4Test.php

```diff
@@ -2795,6 +2795,12 @@ public function test_tracking_opt_out_snippet( $settings, $logged_in, $is_tracki
 		// Remove irrelevant script from throwing errors in CI from readfile().
 		remove_action( 'wp_head', 'print_emoji_detection_script', 7 );
 
+		// Prevent test from failing in CI with deprecation notice.
+		$has_emoji_styles = has_action( 'wp_print_styles', 'print_emoji_styles' );
+		if ( $has_emoji_styles ) {
+			remove_action( 'wp_print_styles', 'print_emoji_styles' );
```

Comment:
This doesn't need to be called conditionally, does it?

Reply:
You're right, it doesn't and is unnecessary. I've removed it, thank you for highlighting!